aws_lambda_powertools/utilities/_data_masking/provider/kms/aws_encryption_sdk.py

```diff
@@ -174,19 +166,35 @@ def decrypt(self, data: str, **provider_options) -> Any:
             ciphertext : bytes
                 The decrypted data in bytes
         """
-        ciphertext_decoded = base64.b64decode(data)
+        try:
+            ciphertext_decoded = base64.b64decode(data)
+        except Error:
+            raise DataMaskingDecryptValueError(
+                "Data decryption failed. Please ensure that you are using a field that was previously encrypted.",
```

Comment:
Maybe we could say something like "Data decryption failed. Please ensure that you are attempting to decrypt data that was previously encrypted." instead of only talking about fields?

Reply:
Great catch @seshubaws! Fixing it now!